pwndbg/commands/nearpc.py

```diff
@@ -97,6 +99,10 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
 
         line   = ' '.join((prefix, "%#x" % i.address, s or '', asm))
 
+        # Highlight the current line if the config is enabled
```

Comment:
This should probably be moved into pwndbg.color.instruction.  This ensures the coloring is applied when invoked internally, instead of just the command.

We should also be sure to only colorize the address and instruction, not ancillary data appended to the right (since this breaks color schemes; bold(blue(foo)) is not necessarily blue).